catalyst/core/callbacks/metrics.py

```diff
@@ -28,7 +28,18 @@ def __init__(
         multiplier: float = 1.0,
         **metrics_kwargs,
     ):
-        """@TODO: Docs. Contribution is welcome."""
+        """
+        Args:
+            prefix (str): key prefix to store computed
```

Comment:
Types in type annotations and docstring are different
Maybe it is better not to include types of args in docstrings at all?